mmda/predictors/lp_predictors.py

```diff
@@ -1,11 +1,12 @@
-from typing import Union, List, Dict, Any, Optional
+from typing import List, Dict, Optional
 
+from PIL.Image import Image
 from tqdm import tqdm
 import layoutparser as lp
 
 from mmda.types import Document, Box, BoxGroup, Metadata
-from mmda.types.names import *
```

Comment:
iirc, the star import was intentional here? or maybe it was intentional somewhere else.